cmd/algorelay/relayCmd.go

```diff
@@ -307,7 +303,11 @@ var updateCmd = &cobra.Command{
 				})
 				anyUpdateError = true
 			} else {
-				fmt.Printf("[%d] OK: %s -> %s:%d\n", relay.ID, relay.Address, name, port)
+				if relay.CheckSuccess {
```

Comment:
If I understand correctly, this output seems to be redundant. 
Similar information is already output in ensureRelayStatus.

Reply:
The `ensureRelayStatus` doesn't always output text, which is why this is needed.